packages/ui-components/src/font.ts

```diff
@@ -0,0 +1,383 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
We don't need this file in this PR now, right? It was just for the ui component?

Reply:
I left the remaining contents of `font.ts` in for the sake of future experiments. But if that's controversial, I'll just remove it (it is true that nothing is currently using it). I want to get this in quick

Reply:
I think it's better to just remove the file entirely. It will still be in the commit history, so easy to resurrect later when it is needed.